### ISSUE TITLE
problem playing demo mp4

### ISSUE THREAD

**Issue by Pepan** [wedesoft/hornetseye-ffmpeg#6]
http://mirrorblender.top-ix.org/movies/sintel-1024-surround.mp4 shows video but audios is like noise.

**Comment by wedesoft** [comment]
Yes, I now get choppy audio here, too. I guess there are changes in FFmpeg and/or ALSA yet again :(

**Comment by wedesoft** [comment]
I think the issue might be that FFmpeg switched from packed to planar audio.
